components/select/index.tsx

```diff
@@ -38,7 +38,7 @@ export interface SelectProps extends AbstractSelectProps {
   defaultValue?: SelectValue;
   mode?: 'default' | 'multiple' | 'tags' | 'combobox';
   optionLabelProp?: string;
-  onChange?: (value: SelectValue) => void;
+  onChange?: (value: SelectValue, option: Object | Object[]) => void;
```

Comment:
`ReactElement<any> | ReactElement<any>[]`

Reply:
@dimitropoulos

Reply:
that was my first inclination, but I did it because the next one looks like this:
```
onSelect?: (value: SelectValue, option: Object) => any;
```

should that change too then?

Reply:
Yes, plz.

Reply:
@yesmeck @afc163 done

Reply:
Thanks!